pandas/parser.pyx

```diff
@@ -582,6 +582,16 @@ cdef class TextReader:
                 else:
                     raise ValueError('Multiple files found in compressed '
                                      'zip file %s', str(zip_names))
+            elif self.compression == 'xz':
+                if PY3:
```

Comment:
there is a doc-string in ``parsers.py`` that needs updating as well as docs in ``io.rst`` (I am pushing a slight fix, so rebase on master)